i18n/ko.toml

```diff
@@ -1,25 +1,73 @@
 # i18n strings for the Korean translation.
+# NOTE: Please keep the entries in alphabetical order when editing
+[announcement_title]
+other = "Black lives matter."
+
+[announcement_message]
+other = "우리는 흑인 공동체를 지지합니다.<br/>인종차별은 용납될 수 없습니다.<br/> 인종차별은 [쿠버네티스 프로젝트의 핵심 가치](https://git.k8s.io/community/values.md)에 상충되며 우리 공동체는 이를 용인하지 않습니다."
```

Comment:
@markruler Sorry for delayed review. I just noticed this PR.
We already reviewed contents in this PR by the merged PR #24346.
But some contents are outdated. So we need to update. 
Please check latest (master) website/i18n/en.toml .
```
[announcement_title]
other = "<img src=\"images/kccnc-na-virtual-2020-white.svg\" style=\"float: right; height: 80px;\" /><a href=\"https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kuberntes.io&utm_medium=search&utm_campaign=KC_CNC_Virtual\">KubeCon + CloudNativeCon NA 2020</a> <em>virtual</em>."

[announcement_message]
other = "4 days of incredible opportunities to collaborate, learn, and share with the entire community!<br />November 17 – 20 2020"
```
https://github.com/kubernetes/website/blob/28c8c2e9b2059612d87b623b06415b041bbf31af/i18n/en.toml#L3

Reply:
@markruler You can ignore this comment !
The banner has been restored from the event banner :)